.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 'stable'
```

Comment:
Can this be a specific node version?

Reply:
Okay, I'll change it to the current node version we're using